package.json

```diff
@@ -13,19 +13,13 @@
   "prettier": "ory-prettier-styles",
   "dependencies": {
     "@openapitools/openapi-generator-cli": "^2.1.18",
-    "grpc-tools": "^1.11.2",
+    "grpc-tools": "^1.11.3",
     "grpc_tools_node_protoc_ts": "^5.3.2"
   },
   "devDependencies": {
     "doctoc": "^2.0.1",
-    "opencollective": "^1.0.3",
     "ory-prettier-styles": "^1.2.0",
     "prettier": "2.7.1",
     "prettier-plugin-packagejson": "^2.2.18"
-  },
-  "collective": {
-    "type": "opencollective",
-    "url": "https://opencollective.com/ory",
-    "logo": "https://opencollective.com/ory/logo.txt"
```

Comment:
Is this needed by anything @aeneasr? The package should not be as far as I can tell.